ll-toggle-card.js

```diff
@@ -0,0 +1,120 @@
+import {html} from '@polymer/lit-element/lit-element.js';
+import {Card} from './ll-card.js';
+import {style} from './ll-toggle-card-styles.js';
+
+/**
+ * `ll-toggle-card`
+ * displays content
+ *
+ * ## Styling
+ *
+ * | Custom property | Description | Default |
+ * | --- | --- | --- |
+ * | `--ll-card-box-shadow` | box shadow for ll-card | `0 0 4px rgba(0, 0, 0, 0.18)` |
+ * | `--ll-card-padding` | padding for ll-card | `calc(var(--lingo-rhythm, 10px) * 3)` |
+ * | `--ll-card-background-color` | background color for ll-card | `#fff` |
+ *
+ * @customElement
+ * @polymer
+ * @demo demo/toggle.html
+ */
+export class ToggleCard extends Card {
+  /**
+   * Declare the properties that will
+   * trigger calls to `render`
+   */
+  static get properties() {
+    return {
+      opened: {type: Boolean},
+    };
+  }
+  /**
+   * Bind local events and set property defaults.
+   */
+  constructor() {
+    super();
+    this.opened = this.getAttribute('opened') !== null;
+    this._open = this._open.bind(this);
+  }
+  /**
+   * Bind click event
+   */
+  connectedCallback() {
+    super.connectedCallback();
+    this.addEventListener('click', this._open);
+  }
+  /**
+   * Remove click event
+   */
+  disconnectedCallback() {
+    super.connectedCallback();
+    this.removeEventListener('click', this._open);
+  }
+  /**
+   * Clicking the card always opens it.
+   */
+  _open() {
+    this.opened = true;
+  }
+  /**
+   * Whether the card is `opened`
+   *
+   * @return {boolean}
+   */
+  get opened() {
+    return this.getAttribute('opened') !== null;
+  }
+  /**
+   * When the card is opened, publish an event to that effect. Update the
+   * element's `opened` attribute in all cases.
+   * @param {boolean} opened - whether to open the card
```

Comment:
```suggestion
   * @param {boolean} triggeredOpenEvent - controls whether to open the card.
```
Not entirely sure what we should name the parameter, but the function name and the parameter are the same. Might be worth changing the parameter name.

Reply:
As a setter, shared method and param name seems like the right call. The only access you have to this (as a setter) is `this.opened = true/false`... so, it's not really a functional construct.